drivers/usb/device/usb_dc_dw.c

```diff
@@ -61,13 +60,6 @@ struct usb_dw_ctrl_prv {
 
 static struct usb_dw_ctrl_prv usb_dw_ctrl;
 
-static inline void _usb_dw_int_unmask(void)
-{
-#if defined(CONFIG_SOC_QUARK_SE_C1000)
-	QM_INTERRUPT_ROUTER->usb_0_int_mask &= ~BIT(0);
```

Comment:
can't we just get the values from the driver here, this should not be QMSI specific.

Reply:
no. The masking of USB interrupts are part of System control subsystem in quark_se.

Reply:
You still can avoid QMSI dependency here. See driver/spi/spi_dw.c, look for _spi_int_unmask()

Reply:
ok got it. I checked _spi_int_unmask() will implement _usb_dw_int_unmask similarly.